content/100-getting-started/02-setup-prisma/100-start-from-scratch/110-relational-databases/100-connect-your-database.mdx

```diff
@@ -3,7 +3,7 @@ title: 'Connect your database'
 metaTitle: 'Connect your database'
 metaDescription: 'Connect your database to your project'
 langSwitcher: ['typescript', 'node']
-dbSwitcher: ['postgres', 'mysql']
+dbSwitcher: ['postgres', 'mysql', 'mssql']
```

Comment:
This should really be `sql-server` or `sqlserver` - `mssql` should not be used anywhere as the identifier. Microsoft is pretty clear, the product is called "SQL Server".

Reply:
Hmm, i originally used sql server, but i noticed that Herve and others made suggestions to change it to Microsoft SQL Server, so i adopted that. What is the correct term here? Because i can make a sweeping change throughout the site

Reply:
Yeah, we got some feedback from MS on this early on: Product name is "SQL Server" or "Microsoft SQL Server", but not "MSSQL". That is also the reason why the feature flag, and existing docs pages do not use the term `mssql` anywhere really.